template/chat.html

```diff
@@ -34,6 +34,9 @@
         }
       },
 
+      /*
+        writes a message in the chat box.
+      */
       sendMessage: function(user, message) {
         if (message != '')
```

Comment:
1. Why would we want to prevent empty messages ? :-D
2. You may rather want something like typeof(message) == "string"
3. The type is going to be converted anyway; if typeof(message) == "number", that'd still work. It all depends on the API contracts we want "sendMessage" to impose.
4. That's not part of the overall changes :P